JavaHackerrank/Arrange_Students.java

```diff
@@ -0,0 +1,92 @@
+import java.io.*;
```

Comment:
Question link : https://www.hackerrank.com/contests/hack-the-interview-iv-apac/challenges/arrange-students/problem